src/test/libradosstriper/rados-striper.sh

```diff
@@ -65,6 +65,10 @@ function run() {
     diff -w $dir/stat $dir/reftrunc || return 1
     
     # test xattrs
+    local notfoundattr="No data available"
```

Comment:
s/notfoundattr/attr_not_found_str/

and can we define this variable at where it is firstly used?